include/secp256k1.h

```diff
@@ -523,6 +523,13 @@ SECP256K1_API int secp256k1_ecdsa_signature_normalize(
  */
 SECP256K1_API extern const secp256k1_nonce_function secp256k1_nonce_function_rfc6979;
 
+/** An implementation of the nonce generation function as defined in BIP-schnorr.
```

Comment:
It's safe indeed to use it with ECDSA, descpite `ecdsa_sign` not providing an `algo16` parameter. But what do you mean by tag?

Reply:
The algo16 parameter isn't observable to users of this function, so it doesn't need to be mentioned.

For the current implementation however, I think you need to document:
* When this function is used in schnorr_sign, it produces bip-schnorr compliant signatures.
* When this function is used in ecdsa_sign, it generates a nonce using an analogue of the bip-schnorr nonce generation algorithm, but with tag "BIPSchnorrNULL" instead of "BIPSchnorrDerive".

And written like that, I think that's a weird choice for the tag, as the fact that ecdsa_sign happens to pass an empty algo16 is an implementation detail that perhaps shouldn't leak into the protocol used - but this is bikeshedding.

Reply:
Ah, ic what you mean. If you have a better tag in mind let me know (not sure what's wrong with BIPSchnorrNULL).

Reply:
added comments